lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -31,14 +31,19 @@
 import collections
 import json
 import re
+import yaml
```

Comment:
@chrisvanheuveln I believe that the CI is failing on yaml, because yaml package might not be installed.
So this needs to be handled with try/except block like this https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/network/aos/aos.py#L43-L47.

Reply:
@trishnaguha Thanks for the tip. I also had to add a workaround check to the UT - this gets us around the shippable requirements but unfortunately means that the UT may only run when we do it manually. Do you have a UT environment that has all the libraries needed to run this? (e.g. with `yaml` and `ordereddict` libs)